### DISCUSSION TITLE
Combo colors in catch should change after every note

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#22885] [Q&A]
In stable, combo colors change after every note in catch.

https://user-images.githubusercontent.com/75596775/225885265-e09212bb-5b63-43d8-bfb6-1340d335912c.mp4

This has been turned off in lazer, so that combo colors now change after every combo. (I say that because it can be replicated to some extent by turning off `Beatmap Colors`.)

https://user-images.githubusercontent.com/75596775/225885897-eb6a386f-7a87-4e1c-b5ce-3888cc15795d.mp4

[Map used in video](https://osu.ppy.sh/b/2620663?m=2)

**Comment by peppy** [comment]
It's kinda weird that it does this. @Walavouchey worth adding this to the differences-between-stable list.

**Comment by frenzibyte** [comment]
See also: https://github.com/ppy/osu/discussions/33520

**Reply by peppy** [reply]
?

**Reply by nagi-desuuu** [reply]
I think they meant https://github.com/ppy/osu/discussions/33520
